.release/linux/postrm

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+rm /opt/nomad/plugins/nomad-driver-podman
```

Comment:
This will let removing the package work without error even if the user has broken the install.

```suggestion
rm -f /opt/nomad/plugins/nomad-driver-podman
```

Reply:
good idea; we should do this on exec2 as well

Reply:
ah good catch